packages/xo-web/src/common/sorted-table/index.css

```diff
@@ -15,3 +15,18 @@
   outline: 2px solid #366e98;
   outline-offset: -2px;
 }
+
+/* Menu item variant */
+.danger{
+    background-color: #990822;
+}
+.primary{
+  background-color: #366e98;
+}
+.success{
+  background-color: #28a745;
+}
+.warning{
+  background-color: #eca649;
+}
+
```

Comment:
Where did you find these colors?

Reply:
Is the current color for `btn-primary`, `btn-success`, ect..
I just used the inspector to find these colors code

Reply:
Why not using bootstrap's `text-*` colors, then?